### PR TITLE
[15.0][MIG] stock_request_tier_validation

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard migration, set manual view = False

### CODE REVIEW

**Review by Saran440 — APPROVED** [review]
Code Review :+1:

File: stock_request_tier_validation/views/stock_request_order_view.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                        name="tier_validated"
+                        string="Validated"
+                        domain="[('validated', '=', True)]"
+                        help="Stock Requests validated and ready to be confirmed"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+</odoo>
```
Comment (OriolMForgeFlow) [inline stock_request_tier_validation/views/stock_request_order_view.xml]: This view should be used in the stock request order model. In Runboat, I can only see two duplicated filters for the stock request model, and there are no filters available for the stock request order model. I am referring to the *Needs My Review* and *Validated* filters.

image

File: stock_request_tier_validation/tests/test_stock_request_tier_validation.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2017-2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo.tests import common
+
+
+class TestStockRequest(common.TransactionCase):
+    def setUp(self):
+        super(TestStockRequest, self).setUp()
```
Comment (OriolMForgeFlow) [inline stock_request_tier_validation/tests/test_stock_request_tier_validation.py]: I think you should change the setUp method to setUpClass according to the [Migration Rules to v15](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0).

image

**Review by OriolMForgeFlow — COMMENTED** [review]
Some small comments.

Functional review: works properly 👍🏼

**Comment by Saran440** [comment]
@OriolMForgeFlow Thank you for your review. I fixed it.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@OriolMForgeFlow Could you update your review ?

**Review by OriolMForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**